src/pat/ajax/tests.js

```diff
@@ -1,6 +1,13 @@
 define(["pat-ajax"], function(pattern) {
     var $lab;
 
+    // Monkey patch Date.now so that we dont get current time mismatches while testing
+    var originalDatenow = Date.now;
+    Date.now = function() {
+        originalDatenow();
+        return "123"; // dummy data instead of current time
+    };
+
```

Comment:
> make check was complaining when that I was declaring it without calling it.

If you restore `Date.now` in `afterEach`, then JSHint won't complain anymore. Just make sure to declare the variable in the scope outside of `beforeEach` and `afterEach`.